demo/demo/security.py

```diff
@@ -0,0 +1,37 @@
+"""
+Dummy security module. Provides fake security functions
+"""
+import bleach
+
+BLEACH_VALID_TAGS = ['p', 'b', 'strong', 'em', 'i', 'u', 'strike', 'ul', 'li',
+                     'ol', 'br', 'span', 'blockquote', 'hr', 'a', 'img', 'h1',
+                     'h2', 'h3', 'h4', 'h5', 'h6', 'table', 'caption', 'th',
+                     'tr', 'td', 'tbody']
+BLEACH_VALID_ATTRS = {
+    'span': ['style', ],
+    'p': ['align', 'style'],
+    'a': ['href', 'rel', 'target', 'name'],
+    'img': ['src', 'alt', 'style'],
+    'div': ['style', ],
+}
+BLEACH_VALID_STYLES = ['color', 'cursor', 'float', 'margin', 'width',
+                       'background-color']
+
+
+def clean(obj):
+    """
+    Use tags from settings to bleach an object.
+    """
+    return bleach.clean(
+        obj,
+        BLEACH_VALID_TAGS,
+        BLEACH_VALID_ATTRS,
+        BLEACH_VALID_STYLES,
+    )
+
+
+def clean_alert(input_string):
```

Comment:
@brunobord what is the purpose of this function ?
I think it should be better to implement a regex-based bleaching method (to identified common XSS payload) to be more effective and more generic.

`bleach` library filters authorized HTML tags, attributes and css-style, then another function like `clean_content()` could detect XSS payload on content that passed `bleach.clean()`.

Reply:
this function belongs to the **demo** site, which is only used in tests. It's here as a "proof of concept" that any 3rd party integrator may use any kind of function they want to clean their own content.

It's also used in this test: https://github.com/peopledoc/django-formidable/pull/378/files#diff-c8e5a0cefee15566bc22894bcf61a925R40-R44 to show how changing the `DJANGO_FORMIDABLE_SANITIZE_FUNCTION` settings and pointing it at an existing function is changing the content passed through it.